tests/security/selinux/selinux_setup.pm

```diff
@@ -34,12 +34,16 @@ sub run {
     }
 
     # install as many as SELinux related packages
-    my $pkgs
-      = "selinux-tools libselinux-devel libselinux1 libselinux1-32bit python3-selinux restorecond mcstrans libsepol1 libsepol-devel libsemanage1 libsemanage-devel checkpolicy setools-console";
-    zypper_call("in $pkgs", timeout => 3000);
-
-    if (is_sle('>=15')) {
```

Comment:
what's gonna happen with < sle 15? do any of this test run on maintenance?

Reply:
> what's gonna happen with < sle 15? do any of this test run on maintenance?

Good question.
SELinux test cases don't run on maintenance atm.
I removed this `'>=15'` as the pkgs should be supported on `<15` products.